nncf/common/graph/graph.py

```diff
@@ -343,9 +351,30 @@ def get_output_edges(self, node: NNCFNode) -> List[NNCFGraphEdge]:
         """
 
         output_nodes = self.get_next_nodes(node)
-        edges = [self.get_edge(node, to_node) for to_node in output_nodes]
+        edges = []
+        for to_node in output_nodes:
+            edges.extend(self._get_edges(node, to_node))
         return sorted(edges, key=lambda x: x.output_port_id)
```

Comment:
Same

Reply:
The point is that _get_edges returns list, and this for statement makes flatten list of edges